src/allotropy/parsers/beckman_vi_cell_blu/vi_cell_blu_parser.py

```diff
@@ -1,98 +1,19 @@
 from __future__ import annotations
 
-from dataclasses import dataclass
-from enum import Enum
-from typing import Any
-
-import numpy as np
-import pandas as pd
-
 from allotropy.allotrope.models.adm.cell_counting.benchling._2023._11.cell_counting import (
-    CellCountingAggregateDocument,
-    CellCountingDetectorDeviceControlAggregateDocument,
-    CellCountingDetectorMeasurementDocumentItem,
-    CellCountingDocumentItem,
-    DataProcessingDocument,
-    DataSystemDocument,
-    DeviceControlDocumentItemModel,
-    DeviceSystemDocument,
-    MeasurementAggregateDocument,
     Model,
-    ProcessedDataAggregateDocument1,
-    ProcessedDataDocumentItem,
-    SampleDocument,
 )
-from allotropy.allotrope.models.shared.definitions.custom import (
-    TQuantityValueCell,
-    TQuantityValueMicrometer,
-    TQuantityValueMillionCellsPerMilliliter,
-    TQuantityValuePercent,
-    TQuantityValueUnitless,
+from allotropy.allotrope.schema_mappers.adm.cell_counting.benchling._2023._11.cell_counting import (
+    Data,
+    Mapper,
 )
-from allotropy.allotrope.models.shared.definitions.definitions import TQuantityValue
-from allotropy.constants import ASM_CONVERTER_VERSION
-from allotropy.exceptions import AllotropeConversionError
 from allotropy.named_file_contents import NamedFileContents
-from allotropy.parsers.beckman_vi_cell_blu.constants import (
-    DEFAULT_ANALYST,
-    DEFAULT_MODEL_NUMBER,
-    VICELL_BLU_SOFTWARE_NAME,
-)
 from allotropy.parsers.beckman_vi_cell_blu.vi_cell_blu_reader import ViCellBluReader
+from allotropy.parsers.beckman_vi_cell_blu.vi_cell_blu_structure import create_data
 from allotropy.parsers.release_state import ReleaseState
-from allotropy.parsers.utils.uuids import random_uuid_str
 from allotropy.parsers.vendor_parser import VendorParser
 
 
-class SampleProperty(Enum):
```

Comment:
double checking, do you think there's value in keeping the enum to be able to view properties at a glance? or does this basically just mirror the ASM so we don't really need it?

also confirming the original value of the Enum was more to be able to use the `data_types` but we don't need it anymore since we pass directly into the ASM dataclass?

(for the record, I am happy to remove it but just making sure it no longer provides value)

Reply:
Yeah - IMHO the body of the `Measurement` creation basically lists out these values.

It would be one thing if adding to this class automatically mapped to the enum (and that may be coming :D) but at this time, it's just another place to make a change.

And yes - the data_type is no longer needed because this is handled by the mapper.